scripts/plugins/extauth-hook

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
why not use `#!/usr/bin/env python3`?

Reply:
When packages RPM will change .py files from '/usr/bin/env python3' to '/usr/bin/python3', but using '/usr/bin/env' might otherwise be preferable, e.g. if used in virtualenvs, where python3 might not be at /usr/bin/python3. https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_shebang_macros

Reply:
I was following other scripts under /etc/xapi.d/plugis, as follows, 
But let's update to use `env`
```grep "/usr/bin/python3" . -rn 
./on-slave:1:#!/usr/bin/python3
./extauth-hook-AD.py:1:#!/usr/bin/python3
./intellicache-clean:1:#!/usr/bin/python3
./tapdisk-pause:1:#!/usr/bin/python3
./testing-hooks:1:#!/usr/bin/python3
./trim:1:#!/usr/bin/python3
./nfs-on-slave:1:#!/usr/bin/python3
./lvhd-thin:1:#!/usr/bin/python3
./extauth-hook:1:#!/usr/bin/python3
./coalesce-leaf:1:#!/usr/bin/python3
```